blender/arm/material/make_mesh.py

```diff
@@ -716,6 +730,13 @@ def make_forward_base(con_mesh, parse_opacity=False, transluc_pass=False):
             frag.write('  , true, spotData.x, spotData.y, spotDir, spotData.zw, spotRight')
         if '_VoxelShadow' in wrd.world_defs and '_VoxelAOvar' in wrd.world_defs:
             frag.write('  , voxels, voxpos')
+        if '_MicroShadowing' in wrd.world_defs:
+            frag.write(' , occspec.x')
+        if '_SSRS' in wrd.world_defs:
+       	    frag.add_include('std/ssrs.glsl')
```

Comment:
What's the reason for including `ssrs.glsl` here? Also this line and line 739 have one space too much indentation (Python is sometimes very nitpicky about this).